deepforest/utilities.py

```diff
@@ -47,10 +47,20 @@ def update_to(self, b=1, bsize=1, tsize=None):
         if tsize is not None:
             self.total = tsize
         self.update(b * bsize - self.n)
-
+def check_new_release():
```

Comment:
I have created this function for `check_new_release` which checks the for any new release.